config/application.rb

```diff
@@ -20,15 +22,12 @@
 module CoredumpWaterSensorApi
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults "6.0"
+    config.load_defaults 6.0
```

Comment:
Did you try loading 6.1 defaults directly? This app is not very complex, so it shouldn't be a problem. If you do so, you can also delete the generated config initializer.

Reply:
I migrated most of the options, but left out the ones which aren't backwards compatible.

Reply:
This app consists only of the api and the admin interface. I'd carelessly opt-in for the defaults until @dbrgn complains (maybe give him a heads up).

Reply:
Well the admin interface currently doesn't work with the 6.1 upgrade due to a assets pipeline issue in production. So I guess I have to rollback...

Reply:
I made a hotfix: #119